shadow-cljs.edn

```diff
@@ -71,6 +71,8 @@
      status-im.config/STATUS_BUILD_PROXY_PASSWORD #shadow/env "STATUS_BUILD_PROXY_PASSWORD"
      status-im.config/STATUS_BACKEND_SERVER_ENABLED #shadow/env "STATUS_BACKEND_SERVER_ENABLED"
      status-im.config/STATUS_BACKEND_SERVER_HOST #shadow/env "STATUS_BACKEND_SERVER_HOST"
+     status-im.config/STATUS_BACKEND_SERVER_MEDIA_SERVER_ENABLE_TLS
+     #shadow/env "STATUS_BACKEND_SERVER_MEDIA_SERVER_ENABLE_TLS"
```

Comment:
Just a reminder to update doc/use-status-backend-server.md because there's a mention to the old env var.